File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    if ineq == ">":
        if eq == "=":
            return 1 if n >= m else 0
        else:
            return 1 if n > m else 0
    else:
        if eq == "=":
            return 1 if n <= m else 0
        else:
            return 1 if n < m else 0
    

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
       boolean answer = false;
        if (ineq.equals(">") && eq.equals("="))
            answer = n >= m;
        else if (ineq.equals("<") && eq.equals("="))
            answer = n <= m;
        else if (ineq.equals(">") && eq.equals("!"))
            answer = n > m;
        else 
            answer = n < m;
        return answer ? 1 : 0;
    }
}